src/pages/NewChairConsole.js

```diff
@@ -10,6 +10,35 @@ import PaginationBar from '../components/PaginationBar';
 import {Storage} from "../libs/storage";
 import ConfirmationModal from "../components/modals/ConfirmationModal";
 
+const wasVoteSubmitted = (vote) => {
+  const targetDate = vote.createDate || vote.updateDate || vote.lastUpdate || vote.lastUpdateDate;
+  return !isNil(targetDate);
+};
+
+const wasFinalVoteSubmitted = (voteData) => {
+  const {type, vote} = voteData;
+  //vote status capitalizes final, election status does not
+  return type === 'FINAL' && !isNil(vote);
+};
+
+const processElectionStatus = (election, votes) => {
```

Comment:
I think the default max value for cyclometric complexity is too low. Most teams use 8-10 as their max value. Is it okay for me to ignore this issue?

Reply:
Absolutely!

Reply:
In fact, this issue is already ignored in codacy, but it still reports as a finding for some reason so this looks like a codacy bug.

Reply:
I re-ran the Codacy action (Bernick gave me access) so the code should be good for review.